documentation/gitbook/how-to-run/before-any-run.md

```diff
@@ -1,98 +1,68 @@
 
 If you are a veteran user, make sure you're signed in on Github.com and through whatever method you use locally on your computer to interact with Github.
 
-## 🧺 Organizing your folders
-
-In order to run any model with flepiMoP, you need access to two separate directories: One containing the flepiMoP code (**code** directory), and another containing the specific input files to run the model of your choosing (and to save the output from that model) (**project** directory). The [flepiMoP code](https://github.com/HopkinsIDD/flepiMoP) is available in a public repository on Github which can be pulled locally to serve as the code directory. We highly recommend also using Github to create your project directory. To get familiar with the code, we recommend starting with our example configurations by making a fork of [flepimop\_sample](https://github.com/HopkinsIDD/flepimop\_sample). If you need to create your own project repository from scratch, see instructions [below](before-any-run.md#create-a-project-repository).
+## 🔐 Access the flepiMoP model code
 
-For both the project repository and flepiMoP code repository, make sure you're on the correct branch, then pull updates from Github. Take note of the local file path to each directory.
+In order to run a model with flepiMoP, you will need to clone the flepiMoP **code** to your machine. 
 
-{% hint style="info" %}
-These directories can be located on your computer wherever you prefer, since you can tell flepiMoP where they are, but we recommend you clone these **flat**, e.g.
+**To clone the `flepiMoP` code repository:**
 
-```
-parentdirectory
-├── flepiMoP
-├── flepimop_sample
-└── project_directory_1
-```
-{% endhint %}
+* If you're using the command line in a terminal, first navigate to the local directory you'll use as the directory for the files that make up `flepiMoP`. Then, use the command: `git clone https://github.com/HopkinsIDD/flepiMoP`
+* If you're using Github Desktop, go File -> Clone Repository, switch to the "URL" tab and copy the URL `https://github.com/HopkinsIDD/flepiMoP` there. For the "Local Path" option, make sure you choose your desired directory.
 
-### 🔐 Access the flepiMoP model code
+You can routinely ensure that your local clone of the flepiMoP code is up to date with upstream `flepiMoP` by navigating in terminal to your `flepiMoP` directory and using the command: `git pull`
 
-To get access to the flepiMoP model code - stored as a [repository on Github.com](https://github.com/HopkinsIDD/flepiMoP) - you need to copy it to your local computer (called "[cloning](../)" in git lingo).
+# Locally install `flepiMoP`
 
-**To clone the `flepimop` code repository**
+First, ensure you have Python and R installed. You need a working Python.7+ installation. We recommend using the latest stable Python release (Python 3.12) to benefit from huge speed-ups and future-proof your installation. We also recommend installing Rstudio to interact with the R code and explore your model outputs.
```

Comment:
I think this is not quite right:

1. We ask that users to install conda, not python/R, which can take care of handling the python/R installation for the appropriate versions, and
2. `flepiMoP` does not support python 3.12 yet, only 3.10 & 3.11. I don't think we state this very well in the documentation anywhere else, but see https://github.com/HopkinsIDD/flepiMoP/blob/6f1f1cbb4d5fc008410ea1c9ac42409ed5b3591b/environment.yml#L8 and https://github.com/HopkinsIDD/flepiMoP/blob/6f1f1cbb4d5fc008410ea1c9ac42409ed5b3591b/.github/workflows/gempyor-ci.yml#L31 There are known issues with 3.12 unfortunately, see GH-244.